dev-util/idea-ultimate/idea-ultimate-14.1.5.141.2735.5_pre.ebuild

```diff
@@ -0,0 +1,41 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
+inherit eutils versionator
+
+SLOT="0"
+PV_STRING="$(get_version_component_range 4-6)" # Needs to be adjusted for each release.
+MY_PV="$(get_version_component_range 1-3)" # Always name EAP-versions with '_pre' for clarity!
+MY_PN="idea"
+
+DESCRIPTION="A complete toolset for web, mobile and enterprise development"
+HOMEPAGE="http://www.jetbrains.com/idea"
+SRC_URI="https://download.jetbrains.com/idea/${MY_PN}IU-${MY_PV}.tar.gz"
+
+LICENSE="IDEA IDEA_Academic IDEA_Classroom IDEA_OpenSource IDEA_Personal"
+IUSE=""
+KEYWORDS="" # No keywords for EAP versions. Code quality sucks.
```

Comment:
devs who require their IDE to function reliable do not use pre-release versions (although the actual programming-code is not influenced by idea), they should have the testing keywords. I did a request for stabilization for the stable version. [561294](https://bugs.gentoo.org/show_bug.cgi?id=561294)